python/rainbow/simulators/prox_rigid_bodies/mass.py

```diff
@@ -339,8 +341,14 @@ def __translate_to_body_space_origin(prop_model):
     prop_body.Iyy = prop_model.Iyy - m * (x * x + z * z)
     prop_body.Izz = prop_model.Izz - m * (x * x + y * y)
     prop_body.Ixy = prop_model.Ixy + m * (x * y)
+
+    too_small = 1.0e-10
+
+    prop_body.Ixy = prop_body.Ixy if np.abs(prop_body.Ixy) > too_small else 0.0
```

Comment:
Should the clamping to zero not be done in line 270-ish were we clamp the values of Ixx, Iyy, and Izz?